addons/mail/models/mail_template.py

```diff
@@ -61,7 +61,9 @@ def default_get(self, fields):
     mail_server_id = fields.Many2one('ir.mail_server', 'Outgoing Mail Server', readonly=False,
                                      help="Optional preferred server for outgoing mails. If not set, the highest "
                                           "priority one will be used.")
-    scheduled_date = fields.Char('Scheduled Date', help="If set, the queue manager will send the email after the date. If not set, the email will be send as soon as possible. You can use dynamic expressions expression.")
+    scheduled_date = fields.Datetime('Scheduled Date',
```

Comment:
Unsure this is really wanted. Help says "dynamic expressions" are supported, which would be broken with a datetime field.